package.json

```diff
@@ -20,22 +20,22 @@
     "webpack-cli": "3.3.5"
   },
   "scripts": {
-    "postinstall": "lerna bootstrap -- --no-package-lock && node scripts/reformat-with-prettier",
-    "bootstrap": "lerna bootstrap -- --no-package-lock && node scripts/reformat-with-prettier",
+    "postinstall": "lerna bootstrap --no-ci -- --no-package-lock && node scripts/reformat-with-prettier",
+    "bootstrap": "lerna bootstrap --no-ci -- --no-package-lock && node scripts/reformat-with-prettier",
     "clean": "lerna run clean",
     "compile": "lerna run --stream compile",
     "lint": "lerna run lint",
     "publish": "node scripts/publish.js",
     "build-change-log": "node scripts/change-log-generator.js",
-    "test": "lerna exec --concurrency 1 --stream --bail=false -- npm run test",
-    "test:unit": "lerna exec --concurrency 1 --stream --bail=false -- npm run test:unit",
-    "test:unit-and-system-tests": "lerna exec --ignore system-tests --concurrency 1 --stream --bail=false -- npm run test:unit && npm run coverage:system-tests",
-    "test:integration": "lerna run test:integration --concurrency 1 --stream",
-    "test:integration-tests-without-system-tests": "lerna exec --ignore system-tests --concurrency 1 --stream --bail=false -- npm run test:integration && npm run test:vscode-integration",
-    "test:vscode-integration": "lerna exec --concurrency 1 --stream --bail=false -- npm run test:vscode-integration",
-    "test:vscode-insiders-integration": "lerna exec --concurrency 1 --stream --bail=false -- npm run test:vscode-insiders-integration",
-    "test:without-system-tests": "lerna exec --ignore system-tests --concurrency 1 --stream --bail=false -- npm run test",
-    "test:system-tests": "lerna run test --scope system-tests --concurrency 1 --stream",
+    "test": "lerna exec --concurrency 1 --stream --bail=false -- npm run test --if-present",
```

Comment:
Running with `--if-present` removes the build errors seen when lerna would try to run npm scripts that are not present in certain modules.